.github/workflows/check_together.yml

```
@@ -41,4 +41,4 @@ jobs:
         elif [[ "${{ steps.check.outputs.files }}" =~ $notexpect ]];then
           echo "::error:: Expected file $notexpect found in ${{ steps.check.outputs.files }}"
           exit 1
-        fi 
\ No newline at end of file
+        fi
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)